strangerstudios/pmpro-affiliates
# Fixes an undefined variable when using autocomplete

### All Submissions:






### Changes proposed in this Pull Request:

Reference to the nonce that is used in the Autocomplete request was undefined and has been fixed

### How to test the changes in this Pull Request:

1. Navigate to Affiliates
2. Add New affiliate and fill out the username (let autocomplete run)
3. Add the user
4. Check the error log, no errors should come up

### Other information:




### Changelog entry

Fixed an undefined variable that was sent during an autocomplete ajax request
